### PR TITLE
Portal 1426/react code cleanup

### PR DESCRIPTION
**Pull request by tbolt**
Small refactors and cleanup for the Dex Portal React App

have all file imports come from the src directory (rather than relative) - makes moving files and refactoring easier (same for the tests directory) - COMPLETED

add TODO for fileSubmissions api call failure to let the user know something went wrong and add default empty string fallback to uploadStatusColor - COMPLETED

add information about MSW to the README - COMPLETED

clean up the main src folder - move screens into Screens/ directory and all components into Components/ - COMPLETED

### CODE REVIEW

**Review by tbolt — COMMENTED** [review]
Looks good to me. @sublettk or @ian-sawyer can you do a final approval for this?

File: src/screens/Submissions.tsx [hunk 1]
```diff
@@ -38,10 +39,17 @@ function Submissions() {
...
+    switch (status) {
+      case "Uploading":
+        return "busy";
+      case "Uploaded":
+        return "success";
+      case "Failed":
+        return "error";
+      default:
+        return "";
+    }
```
Comment (sublettk) [inline src/screens/Submissions.tsx:52]: Let's keep the original implementation.

**Review by sublettk — CHANGES_REQUESTED** [review]
LGTM!

File: src/screens/Submissions.tsx (hunk 1 above)
Comment (ZacharyRizer) [reply src/screens/Submissions.tsx]: Switched back.

**Review by sublettk — APPROVED** [review]

**Pull request merged**